kuma/api/v1/plus/notifications.py

```diff
@@ -229,9 +232,8 @@ class UpdateWatch(Schema):
     update_custom_default: bool = False
 
 
-@watch_router.post("/watch{path:raw_url}", response={200: Ok, 400: NotOk, 400: NotOk})
-def update_watch(request, raw_url, data: UpdateWatch):
-    url = DocumentURL.normalize_uri(raw_url)
+@watch_router.post("/watching/", response={200: Ok, 400: NotOk, 400: NotOk})
+def update_watch(request, url: str, data: UpdateWatch):
```

Comment:
Just maybe double check that we don't have any url encode issues / that the `normalize_uri` did nothing crucial.